src/main/java/com/google/sps/servlets/GoodDeedsServlet.java

```diff
@@ -68,6 +69,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
         String description = getParameter(request, DESCRIPTION, DEFAULT_VALUE);
         String link = getParameter(request, LINK, DEFAULT_VALUE);
         long timestamp = System.currentTimeMillis();
+        Collection<String> comments = new ArrayList<>();
```

Comment:
same question here, can we just use a `List`?

Reply:
Sure!